webrender/src/renderer.rs

```diff
@@ -751,30 +756,57 @@ impl<B: hal::Backend> GpuCacheTexture<B> {
         // Create the new texture.
         assert!(height >= 2, "Height is too small for ANGLE");
         let new_size = DeviceIntSize::new(MAX_VERTEX_TEXTURE_WIDTH as _, height);
-        #[cfg(feature="gleam")]
         let rt_info = Some(RenderTargetInfo { has_depth: false });
-        #[cfg(not(feature="gleam"))]
-        let rt_info = None;
-        let mut texture = device.create_texture(
-            TextureTarget::Default,
-            ImageFormat::RGBAF32,
-            new_size.width,
-            new_size.height,
-            TextureFilter::Nearest,
-            rt_info,
-            1,
-        );
 
-        // Blit the contents of the previous texture, if applicable.
-        if let Some(blit_source) = blit_source {
-            device.blit_renderable_texture(&mut texture, &blit_source);
-            device.delete_texture(blit_source);
+        #[cfg(not(feature = "gleam"))]
+        {
+            let mut texture = device.create_gpu_cache_texture(
+                TextureTarget::Default,
+                ImageFormat::RGBAF32,
```

Comment:
most parameters here are useless for this method

Reply:
Removed the useless parameters in https://github.com/szeged/webrender/commit/99969449fecfb4a124cfdcb88684c4bf8f41e4e8#diff-f5062b694b9fe53fc1757ed483d577d9R762